public/whiteboard/js/Brush.js

```diff
@@ -1,11 +1,11 @@
 class Brush {
-  constructor (img, p5) {
+  constructor (img, link, p5) {

```

Comment:
i modified the Brush class slightly so we wouldn't need outer scope variables like `img` or `link`

Reply:
later updates modified this back anyway